packaging/installer/methods/kubernetes.md

```diff
@@ -90,9 +90,24 @@ services](https://github.com/netdata/helmchart#service-discovery-and-supported-s
 If you haven't changed listening ports or other defaults, service discovery should find your pods, create the proper
 configurations based on the service that pod runs, and begin monitoring them immediately after depolyment.
 
-However, if you have changed some of these defaults, you'll need to open the `netdata-helmchart/sd-child.yml` file and
-edit the ports or image names that service discovery uses to find supported pods. You can then run `helm upgrade netdata
-./netdata-helmchart` to push changes to your cluster.
+However, if you have changed some of these defaults, you'll need to copy the `netdata-helmchart/sdconfig/child.yml`
+file, edit it, and pass the changed file to `helm install`. First, copy the file to a new location.
+
+```bash
+cp netdata-helmchart/sdconfig/child.yml .
+```
+
+Edit the new `child.yml` file according to your needs. See the [Helm chart
+configuration](https://github.com/netdata/helmchart#configuration) and the file itself for details. You can then run
+`helm install`/`helm upgrade` with the `--set-file` argument to use your configured `child.yml` file instead of the
+default.
+
+```bash
+helm install --set-file sd.child.configmap.from.value=../child.yml netdata ./netdata-helmchart
+helm upgrade --set-file sd.child.configmap.from.value=../child.yml netdata ./netdata-helmchart
```

Comment:
> `../child.yml`

this should be path to the file config, i used `../child.yml` a an example and to show that there is no requirement to have the file in the root dir.

Reply:
something like this

```cmd
helm install --set-file sd.child.configmap.from.value=<FILE> netdata ./netdata-helmchart
```

---

```
--set-file stringArray  set values from respective files specified via the command line (can specify multiple or separate values with commas: key1=path1,key2=path2)
```

Reply:
Change the instructions to copy `child.yml` outside the helmchart root directory, and tested it myself. Works great!

![image](https://user-images.githubusercontent.com/1153921/86296946-eb9f2300-bbae-11ea-97f5-9c5c7a9a4c45.png)

Changed their names to include `-testing-`.